app/helpers/candidates_helper.rb

```diff
@@ -17,102 +17,143 @@ def candidate_display_activity(event)
     actor_account_name = compose_actor_account_name(event)
 
     text =
-      if event.type == "email_received"
-        []
-      else
-        [actor_account_name]
-      end
-    text <<
       case event.type
       when "candidate_added"
-        "added the candidate"
+        if event.properties["method"] == "manual" && event.actor_account_id.present?
```

Comment:
Can we just add another method if candidate applies instead of this condition? it could be "applied" or "career site".
With this code it's unclear why it was added, also, later it will help build stats.